typescript/src/deposit.ts

```diff
@@ -42,16 +41,20 @@ export interface Deposit {
   blindingFactor: string
 
   /**
-   * Compressed (33 bytes long with 02 or 03 prefix) Bitcoin public key of
-   * the wallet that is meant to receive the deposit.
+   * Public key hash of the wallet that is meant to receive the deposit. Must
+   * be an unprefixed hex string (without 0x prefix).
+   *
+   * You can use `computeHash160` function to get the hash from a plain text public key.
    */
-  walletPublicKey: string
+  walletPubKeyHash: string
 
   /**
-   * Compressed (33 bytes long with 02 or 03 prefix) Bitcoin public key that
-   * is meant to be used during deposit refund after the locktime passes.
+   * Public key hash that is meant to be used during deposit refund after the
+   * locktime passes. Must be an unprefixed hex string (without 0x prefix).
+   *
+   * You can use `computeHash160` function to get the hash from a plain text public key.
    */
-  refundPublicKey: string
+  refundPubKeyHash: string
```

Comment:
Let's be consistent with naming and avoid abbreviations:
```suggestion
  refundPublicKeyHash: string
```
Remember about changing all references as well

Reply:
Changed it to `PublicKey`:
fcc0e42e55b0fde1659bac5dd7ac081241f0cdd1
69dfd10cfee4a7b56dae13a76a9600da11ff171f